From 64eb31d02d4933ad414239130df3dff45a0f6f91 Mon Sep 17 00:00:00 2001
From: Serge Hallyn <serge@hallyn.com>
Date: Fri, 11 Nov 2022 18:20:37 -0600
Subject: [PATCH 1/1] src/lxc/meson.build: fix the static library path

Since switching to meson, liblxc.a is being shipped as liblxc_static.a.
Change it back to liblxc.a.

Signed-off-by: Serge Hallyn <serge@hallyn.com>
---
 src/lxc/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lxc/meson.build b/src/lxc/meson.build
index b4609e203..6d4c70097 100644
--- a/src/lxc/meson.build
+++ b/src/lxc/meson.build
@@ -149,7 +149,7 @@ if want_selinux and libselinux.found()
 endif
 
 liblxc_static = static_library(
-    'lxc_static',
+    'lxc',
     liblxc_sources + include_sources + netns_ifaddrs_sources,
     install: true,
     include_directories: liblxc_includes,
-- 
2.37.2

